src/csharp/Microsoft.Spark/ML/Param/Param.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+
+namespace Microsoft.Spark.ML.Feature.Param
+{
+    /// <summary>
+    /// A <see cref="Param"/> with self-contained documentation and optionally default value.
+    ///
+    /// A <see cref="Param"/> references an individual parameter that includes documentation, the
+    /// name of the parameter and optionally a default value. Params can either be set using the
+    /// generic <see cref="Param"/> methods or by using explicit methods. For example
+    /// <see cref="Bucketizer"/> has SetHandleInvalid or you can call GetParam("handleInvalid")
```

Comment:
Fix line length if you add this change.
```suggestion
    /// <see cref="Bucketizer"/> has <c>SetHandleInvalid</c> or you can call <c>GetParam("handleInvalid")</c>
```